### PR TITLE
Fix minor issues in the tvmc tune CLI

### PR DESCRIPTION
**Pull request by nodeav**
Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

File: python/tvm/driver/tvmc/autotuner.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def add_tune_parser(subparsers):
         help="the host compilation target, defaults to 'llvm'",
         default="llvm",
     )
-    parser.add_argument("--timeout", default=10, help="compilation timeout, in seconds")
+    parser.add_argument("--timeout", default=10, help="compilation timeout, in seconds", type=int)
```
Comment (comaniac) [inline python/tvm/driver/tvmc/autotuner.py:119]:
```suggestion
    parser.add_argument("--timeout", type=int, default=10, help="compilation timeout, in seconds")
```

**Review by comaniac — APPROVED** [review]
Otherwise LGTM

**Review by jwfromm — APPROVED** [review]

File: python/tvm/driver/tvmc/autotuner.py (hunk 1 above)
Comment (nodeav) [reply python/tvm/driver/tvmc/autotuner.py]: done

**Review by leandron — APPROVED** [review]
@nodeav thanks for the fix. LGTM

**Review by comaniac — APPROVED** [review]

**Comment by comaniac** [comment]
Thanks @nodeav @leandron @jwfromm

**Pull request merged**